admin-dev/themes/default/scss/partials/_header.scss

```
@@ -574,6 +689,27 @@ $search-border-color: #bbcdd2;
     border-radius: 0;
     box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.1);
 
+    @include media-breakpoint-down(sm) {
+      left: 0;
+      right: inherit;
+      // stylelint-disable-next-line
+      width: 100vw!important;
+      top: 60px;
+      border: none;
+      border-top: 1px solid #EAEBEC;
```

```suggestion
      border-top: 1px solid $background-grey;
```
Even if I dislike adding color in variable name ^^